src/SingleMovie/SingleMovie.js

```diff
@@ -1,31 +1,90 @@
-import React from 'react';
-import '../SingleMovie/SingleMovie.css'
+import React, { Component } from 'react';
+import '../SingleMovie/SingleMovie.css';
+import { Link } from 'react-router-dom';
 
-const SingleMovie = ({ movie, switchState }) => {
-if(!movie) {
-    return null
+class SingleMovie extends Component {
+  constructor() {
+    super();
+    this.state = {
+      movie: {},
```

Comment:
Thanks for changing this to an empty object, we never want null within our state